### ISSUE TITLE
Crash when reading mp3 file

### ISSUE THREAD

**Issue by dzek69** [Borewit/music-metadata#828]
**Bug description**
JS Error is thrown while reading file that seems valid to me and all apps I'm using it with (including Windows Explorer)

```
/Projects/ytmsync/node_modules/music-metadata/lib/mpeg/XingTag.js:58
        const majorMinorVersion = xingInfoTag.lame.version.match(/\d+.\d+/g)[0]; // e.g. 3.97
                                                                            ^
TypeError: Cannot read property '0' of null
    at Object.readXingHeader (/Projects/ytmsync/node_modules/music-metadata/lib/mpeg/XingTag.js:58:77)
    at MpegParser.readXingInfoHeader (/Projects/ytmsync/node_modules/music-metadata/lib/mpeg/MpegParser.js:489:25)
    at MpegParser.readXtraInfoHeader (/Projects/ytmsync/node_modules/music-metadata/lib/mpeg/MpegParser.js:452:33)
    at MpegParser.skipSideInformation (/Projects/ytmsync/node_modules/music-metadata/lib/mpeg/MpegParser.js:441:9)
    at MpegParser.parseAudioFrameHeader (/Projects/ytmsync/node_modules/music-metadata/lib/mpeg/MpegParser.js:368:13)
    at MpegParser._parse (/Projects/ytmsync/node_modules/music-metadata/lib/mpeg/MpegParser.js:233:24)
    at MpegParser.parseID3v2 (/Projects/ytmsync/node_modules/music-metadata/lib/id3v2/AbstractID3Parser.js:41:9)
    at MpegParser.parse (/Projects/ytmsync/node_modules/music-metadata/lib/id3v2/AbstractID3Parser.js:24:13)
    at Function._parse (/Projects/ytmsync/node_modules/music-metadata/lib/ParserFactory.js:161:9)
    at parseFile (/Projects/ytmsync/node_modules/music-metadata/lib/index.js:46:16)
```

**Expected behavior**
No crash

**Audio file demonstrating the problem**
[Dinosaurs Will Die (live).zip](https://github.com/Borewit/music-metadata/files/6566958/Dinosaurs.Will.Die.live.zip)

**Comment by Borewit** [comment]
Exception thrown due to invalid Xing version tag. Xing version set to `ZyK! `.
I also marked it as bug, because I think [music-metadata](https://github.com/Borewit/music-metadata) should be able to handle that.

**Comment by Borewit** [comment]
Should be fixed in [v7.8.6](https://github.com/Borewit/music-metadata/releases/tag/v7.8.6).

Thanks for reporting the issue @dzek69!
